pom.xml

```diff
@@ -1,295 +1,297 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <name>FWU Keycloak extensions</name>
-    <description>Keycloak extensions for customizations of the FWU Keycloak</description>
+  <name>FWU Keycloak extensions</name>
+  <description>Keycloak extensions for customizations of the FWU Keycloak</description>
 
-    <groupId>de.intension</groupId>
-    <artifactId>fwu-kc-extensions</artifactId>
-    <version>${revision}</version>
-    <packaging>pom</packaging>
+  <groupId>de.intension</groupId>
+  <artifactId>fwu-kc-extensions</artifactId>
+  <version>${revision}</version>
+  <packaging>pom</packaging>
 
-    <modules>
-        <module>remove-user-on-logout</module>
-        <module>hmac-mapper</module>
-        <module>whitelist-authenticator</module>
-        <module>multi-value-user-attribute-mapper</module>
-        <module>whitelist-authenticator-schools</module>
-        <module>user-info-api</module>
-        <module>user-info-requester</module>
-        <module>user-info-provider</module>
-        <module>acronym-mapper</module>
-        <module>school-mapper</module>
-        <module>login-event-publisher</module>
-        <module>account-linking-authenticator</module>
-    </modules>
+  <modules>
+    <module>remove-user-on-logout</module>
+    <module>hmac-mapper</module>
+    <module>whitelist-authenticator</module>
+    <module>multi-value-user-attribute-mapper</module>
+    <module>whitelist-authenticator-schools</module>
+    <module>user-info-api</module>
+    <module>user-info-requester</module>
+    <module>user-info-provider</module>
+    <module>acronym-mapper</module>
+    <module>school-mapper</module>
+    <module>login-event-publisher</module>
+    <module>account-linking-authenticator</module>
+  </modules>
 
-    <properties>
-      <revision>4.10.2</revision>
-      <keycloak.version>22.0.5</keycloak.version>
-      <java.version>17</java.version>
-        <maven.compiler.source>${java.version}</maven.compiler.source>
-        <maven.compiler.target>${java.version}</maven.compiler.target>
-        <maven.compiler.release>${java.version}</maven.compiler.release>
-        <mockito.version>4.9.0</mockito.version>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
+  <properties>
+    <revision>4.10.2</revision>
```

Comment:
Increase Version Please. And Also add Changelog.

Reply:
Done!! thanks for noticing